docs/content/en/docs/policy-library/observability/_index.md

```diff
@@ -30,95 +33,170 @@ description: >
 
 ### Example jq Filter
 
 ```shell
-jq 'select(.process_loader != null) | "\(.time) \(.process_loader.process.pod.namespace) \(.process_loader.process.binary) \(.process_loader.process.arguments) \(.process_loader.path)"
+jq 'select(.process_exec != null) | select(.process_exec.process.binary_properties != null) | select(.process_exec.process.binary_properties.setuid != null or .process_exec.process.binary_properties.setgid != null) | "\(.time) \(.process_exec.process.pod.namespace) \(.process_exec.process.pod.name) \(.process_exec.process.binary) \(.process_exec.process.arguments) uid=\(.process_exec.process.process_credentials.uid) euid=\(.process_exec.process.process_credentials.euid)  gid=\(.process_exec.process.process_credentials.gid) egid=\(.process_exec.process.process_credentials.egid) binary_properties=\(.process_exec.process.binary_properties)"'
 ```
 
 ### Example Output
 
 ```shell
-"2023-10-31T19:42:33.065233159Z default/xwing /usr/bin/curl https://ebpf.io /usr/lib/x86_64-linux-gnu/libssl.so.3"
+"2024-02-05T20:20:50.828208246Z null null /usr/bin/sudo id uid=1000 euid=0  gid=1000 egid=1000 binary_properties={\"setuid\":0,\"privileges_changed\":[\"PRIVILEGES_RAISED_EXEC_FILE_SETUID\"]}"
+"2024-02-05T20:20:57.008655978Z null null /usr/bin/wall hello uid=1000 euid=1000  gid=1000 egid=5 binary_properties={\"setgid\":5}"
+"2024-02-05T20:21:00.116297664Z null null /usr/bin/su --help uid=1000 euid=0  gid=1000 egid=1000 binary_properties={\"setuid\":0,\"privileges_changed\":[\"PRIVILEGES_RAISED_EXEC_FILE_SETUID\"]}"
 ```
 
-## Binary Execution in /tmp {#tmp-execs}
+## Privileges Escalation via File Capabilities Execution {#privileges-fscaps}
 
 ### Description
 
-Monitor execution of a binary in the /tmp directory.
+Monitor execution of binaries with file capabilities.
 
 ### Use Case
 
-Preventing execution of executables in `/tmp` is a common best-practice as several canned exploits rely on writing and then executing malicious binaries in the `/tmp` directory. A common best-practice to enforce this is to mount the `/tmp` filesystem with the `noexec` flag. This observability policy is used to monitor for violations of this best practice.
+File capabilities allow the binary execution to acquire more privileges
+to perform specific tasks. They are stored in the extended attribute part
+of the binary on the file system. They can be set using the [setcap](https://man7.org/linux/man-pages/man8/setcap.8.html) tool.
+
+For further reference, please check [capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html)
+man page; section `File capabilities`.
+
+Detecting the execution of `file capabilities` binaries is a common
+best-practice, since they cross privilege boundaries which make them a suitable
+target for attackers. Such binaries are also used by attackers to hide their
+privileges after a successful exploit.
+
+### Requirement
+
+Run Tetragon with `enable-process-creds` setting set to enable visibility
```

Comment:
The same comment applies here. This level of configuration details doesn't belong to the "Policy Library".